docker/Dockerfile-tests

```diff
@@ -1,25 +1,18 @@
-FROM fedora:30
+FROM registry.access.redhat.com/ubi8/ubi
```

Comment:
same comment.
BTW, have you considered using ubi-minimal image?

Reply:
I'll give it a shot.